docs/extensions/dnsrecord.md

```diff
@@ -127,10 +127,20 @@ If this feature gate is enabled, all three DNS records mentioned above (internal
 These providers can be used for `DNSEntry` resources needed by workloads deployed on the shoot cluster.
 
 If the feature gate is disabled, Gardener will not create any `DNSRecord` resources and use `DNSProvider` / `DNSEntry` resources for its DNS records. 
-Currently, the feature gate is still in `Alpha` stage and is therefore disabled by default.
+The feature gate is enabled by default starting from `v1.39`.
```

Comment:
I am missing information about the Gardener version that adds the support for DNSResources. Is this information available somewhere?

Reply:
```suggestion
The feature gate was introduced in `v1.27` and was in `Alpha` stage (disabled by default) until `v1.39`. With `v1.39` the feature gate is graduated to `Beta` and it is enabled by default.
```